scripts/scale_test.py

```diff
@@ -0,0 +1,29 @@
+import random
+import time
+from locust import HttpUser, between, task
+
+random.seed(time.time())
+
+
+class WebsiteUser(HttpUser):
+    wait_time = between(1, 2)
```

Comment:
If we want the test to run faster we modify or remove this value.

Reply:
I'm just running the test for 5 minutes